src/azure-cli/azure/cli/command_modules/containerapp/_clients.py

```diff
@@ -16,7 +16,7 @@
 
 logger = get_logger(__name__)
 
-CURRENT_API_VERSION = "2024-03-01"
+CURRENT_API_VERSION = "2024-10-02-preview" # "2024-03-01"
```

Comment:
@Tratcher 
For preview feature, please use extension: https://github.com/Azure/azure-cli-extensions